src/worker.ts

```diff
@@ -28,15 +28,15 @@ const browserIsProbablySafari = browser.satisfies({
 const wasmPromise = fetch("/assets/wormhole.wasm");
 let rpc: RpcProvider | undefined = undefined;
 
-const bufferSize = 1024 * 4 // 4KiB
+const bufferSize = 16384 // to match the sending side buffersize
```

Comment:
`bufferSize` is only used in `handleReceiveFileData` on line 110. I'm not sure I understand this change.

The size of the send-side buffer is determined by the size of the `ArrayBuffer` that's passed to [`FileWrapper#Read`](https://github.com/LeastAuthority/wormhole-william/blob/master/wasm/client.go#L127).

If we want to change the size of the receive-side buffer we currently also need to change [client.go:345](https://github.com/LeastAuthority/wormhole-william/blob/master/wasm/client.go#L345).